### ISSUE TITLE
Feature Request: Embed Persistent Auto Updating Status

### ISSUE THREAD

**Issue by SolidSnake916** [Therosin/MisCord#23]
I want to suggest a special ember you can have posted to a Status Channel, which can display certain parameters.
- Must have: Server Name, Map Name
- Wipe Information: like how many days, hours, minutes to a wipe.
- Players Online: Shows 30/100 and refreshes often.
- Next Restart: Shows how many hours and minutes remain until a restart.
- Weather Pattern: Maybe?
- Time of Day: Maybe?
- Steam Connect Link to launch and auto join server.

basically m>server-info but auto updating and nicer :)
